src/components/TokenEnterAmount.tsx

```diff
@@ -28,13 +29,18 @@ import { Spacing } from 'src/styles/styles'
 import variables from 'src/styles/variables'
 import { type TokenBalance } from 'src/tokens/slice'
 import { convertLocalToTokenAmount, convertTokenToLocalAmount } from 'src/tokens/utils'
+import { NetworkId } from 'src/transactions/types'
 import { parseInputAmount } from 'src/utils/parsing'
 
 export const APPROX_SYMBOL = '≈'
 
 const BORDER_RADIUS = 12
 export const FETCH_UPDATED_TRANSACTIONS_DEBOUNCE_TIME_MS = 250
 
+export function getDisplayTokenName<T extends { symbol: string; networkId: NetworkId }>(token: T) {
+  return `${token.symbol} on ${NETWORK_NAMES[token.networkId]}`
+}
+
```

Comment:
This was moved to a exportable function just for the sake of being able to use it in the `SwapScreenV2.test.tsx` file as well.

Reply:
not related to the actual change, but it will show "on" without translation no matter what language is used? are we okay with that?